### DISCUSSION TITLE
Incorrect Zeus watchface template for 2.0

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#223] [Developer Feedback]
Hello.
With zeus-cli 1.3.0-beta.10 default watchface template targeting ZeppOS 2.0 doesn't complain with zeus checks.

Steps to reproduce:

- zeus init
- zeus build

![Screenshot from 2023-07-19 06-36-37](https://github.com/zepp-health/zeppos-forum/assets/5894573/64bc23c3-82e9-4266-839c-184ef4d20c50)

**Comment by htoooth** [comment]
@kartun83 Are you referring to a warning? Here, warning means that the icon provided by the project does not meet the design standards. It seems that there is an issue with the template we provided. We will update it accordingly. Thank you.

**Reply by kartun83** [reply]
yes, I meant that template is outdated.

**Comment by htoooth** [comment]
@kartun83 This warning does not affect development and packaging. You can ignore it.
